server/views/pages/create/additionalLicenceConditionInput.njk

```diff
@@ -15,9 +16,24 @@
         </div>
     </div>
 
-    <form method="POST">
+    {# Check for any fileUpload type in the inputs for this condition - it changes tohe form submission #}
+    {% set fileUpload = false %}
+    {% for input in config.inputs %}
+       {% if input.type === 'fileUpload' %}
+           {% set fileUpload = true %}
+       {% endif %}
+    {% endfor %}
+
+    {# Alter the form encoding to multi-part formdata and supply csrf token in query params for file upload conditions #}
+    <form method="POST"
+        {% if fileUpload %}
+            encType="multipart/form-data"
+            action="?_csrf={{ csrfToken }}{{ fromReviewAppend }}"
```

Comment:
Can be the following? Then you dont need the extra `fromReviewAppend`

`action="{{fromReview}}&_csrf={{csrfToken}}"`

Reply:
Thanks. Will take a look now.

Reply:
No, it needs the `?` prior to the query parameters. The `_csrf` param is always needed in the form action for file uploads, but the fromReview is optional.